spark/v4.0/build.gradle

```diff
@@ -181,14 +182,15 @@ project(":iceberg-spark:iceberg-spark-extensions-${sparkMajorVersion}_${scalaVer
     testRuntimeOnly libs.jetty.servlet
     testRuntimeOnly libs.sqlite.jdbc
 
+    testImplementation 'com.jolbox:bonecp:0.8.0.RELEASE'
     testImplementation libs.avro.avro
     testImplementation libs.parquet.hadoop
     testImplementation libs.awaitility
-    testImplementation "org.apache.datafusion:comet-spark-spark${sparkMajorVersion}_${scalaVersion}:0.5.0"
+    testImplementation "org.apache.datafusion:comet-spark-spark3.5_2.13:0.5.0"
```

Comment:
Why are we using the old version of comet here? Just because the new one isn't available?

Reply:
Rignt, we don't have a Spark 4.0 Comet yet.